docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,24 +2152,744 @@ Drop all values in mods:part.
+        </role>
+      </name>
+    </relatedItem>
+    <relatedItem type="constituent">
+      <titleInfo>
+        <title>1963 Missouri Basin Project weekly report, Party no.11 - Moreau Party, Report no.2-11, June 21-August 30, 1963</title>
+      </titleInfo>
+      <name authority="naf" valueURI="http://id.loc.gov/authorities/names/no2004118055">
+        <namePart>Mallory, Oscar L.</namePart>
+        <role>
+          <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+      </name>
+    </relatedItem>
+    <relatedItem type="constituent">
+      <titleInfo>
+        <title>University of Kansas Milford Reservoir Archeological Party, Cooperators Party A, Report no.3, June 28, 1963</title>
+      </titleInfo>
+      <name>
+        <namePart>Schock, Jack</namePart>
+        <role>
+          <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+      </name>
+    </relatedItem>
+    <relatedItem type="constituent">
+      <titleInfo>
+        <title>Nebraska State Historical Society - National Science Foundation Logan Creek Project - Cooperators Party C, Report no.1, June 28, 1963</title>
+      </titleInfo>
+      <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n88243079">
+        <namePart>Kivett, Marvin F.</namePart>
+        <role>
+          <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+      </name>
+    </relatedItem>
+    <relatedItem type="constituent">
+      <titleInfo>
+        <title>1963 Missouri Basin Project Weekly report, Party no. 9 - Chapelle Creek, Report no.3-10, July 5-August 23, 1963</title>
+      </titleInfo>
+      <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n78078895">
+        <namePart>Folan, William J.</namePart>
+        <role>
+          <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+      </name>
+    </relatedItem>
+    <relatedItem type="constituent">
+      <titleInfo>
+        <title>1963 Missouri Basin Project weekly report, Party no. 4 - Garrison Diversion, Report no.1-6, July 26-August 30, 1963</title>
+      </titleInfo>
+      <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n50038965">
+        <namePart>Johnson, Elden</namePart>
+        <role>
+          <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+      </name>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
```

Comment:
I don't recall what is correct here... **http://id.loc.gov/vocabulary/relators/** or **http://id.loc.gov/vocabulary/relators**?

Reply:
The reason why I ask is that we (or maybe just I did) seem to do this differently across different top-level elements. I don't recall how to question and ask this using RDF inquisitor.

Reply:
It should be:  `http://id.loc.gov/vocabulary/relators/`

No need to use inquisitor. You're prefixing the URI so that you don't have to write everything out like you would in the old n3 way.  The old n3 way would look something like this:

```
<https://example.org/objects/1> <http://id.loc.gov/vocabulary/relatorscre> <http://id.loc.gov/authorities/names/n83189337> .
```

Look at the predicate or try to resolve in a browser.  You'll get a 404 not found.

Make sense?

Reply:
If I had 5¢ for every time I've left a trailing slash off of a namespace URI, I would be able to retire and live a life of profligate excess. 

@mathewjordan thanks for catching this!